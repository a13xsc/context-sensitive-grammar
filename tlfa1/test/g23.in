     (   {         " , D          ,    R   ,         ;  ,         S   , b  ,      1         ,      X  ,         w        ,        T          , 7          ,   t        ,      A,    Z , W       ,q,    o   ,   L  ,   U  ,        P      ,        ~       , n    ,          B          ,      6         ,       E,     C          ,    r    ,        5    ,!     ,      N ,I     ,         G    ,        Q      ,      -     , `        ,',  V       ,          2  ,O  ,   M         , H  ,  J      ,       Y         ,      F        }       ,       {    ;    ,        "  ,        `     ,     2   ,-        ,   7         ,        n  ,     b       ,  1 ,   r        ,         ~    ,6        ,         !      ,       o  ,     w      ,   t  ,   ',  5          , q               }              ,           {      (  C     ,2756!  ),      (C   ,         P     )    ,         (        C      ,          EFo6YoVGJW5~EPPb  )      ,        ( B ,  M1;UbTC!2XVNYqD-PMW  )        ,     (        D          ,   S          ),(     F ,         wrHtItXHFP  ),     (         H    ,         MEHLT6"AqDFY'7Z ) ,     (          J      ,     WWWWWWWWWWWWWWWWWWWW   )      ,         (L          ,          JWJ       )          ,(         Q      ,       IA~LqYoFPQQDrI25        )      ,     (     P     ,     2~57!1nrq`tt      )   ,    (    R  ,         OVMb"6DQnFXU )   ,        (     R      ,       WJJWW  )       ,        (   R    ,         C'XJX!QLwWLXPoL-DS!T     ),      (U     ,    Q`n1o-tIRSnLL   )    ,  ( U   ,       LLRRLR          )        ,         (         T       ,`'';5t"7~5wq'~2!1-    )  ,         ( T      ,ZVG17XOG  )          ,   (       W   ,GSU        ),     (       W          ,e          )      ,(     W        ,   B;1;TV`~QoDMI`Q6P~wJ      )          ,  (   X       ,"1QT~          )          ,   (    X       ,        2276512";;652t~t`not     ) ,       (       Z     ,   "FI~NB~CSV'CPn  )        ,  (     Z ,         n'-162"r1ot   )            }       ,  S    )   
