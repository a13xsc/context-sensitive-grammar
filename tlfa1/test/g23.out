D
B
Q
E
I
F
M
H
N
G
S
Y
O
V
A
